index.html

```diff
@@ -198,9 +198,9 @@ <h2>About me</h2>
     <section class="footer">
         <h2>I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</h2>
         <form action="" class="footer-form">
```

Comment:
- [ ] Kindly add to your form tag, the properties missing. These are two.

`action=https://formspree.io/f/{form_id}`
`method= POST`